modules/managers/galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.sdv.manager/src/main/java/dev/galasa/sdv/internal/SdvUserImpl.java

```diff
@@ -136,7 +136,7 @@ public void setNotRecording() {
     private void waitForLoggedOnText(ICicsTerminal cicsTerminal) throws CicstsManagerException {
 
         String[] pass = {"Sign-on is complete"};
-        String[] fail = {"Your password has expired. Please type your new password.",
+        String[] fail = {"Your password has expired. Type your new password.",
```

Comment:
Same here - this could come from the server, so I wonder if changing this could cause issues?

Reply:
Reverted